doc/ref/chap-type-method.md

```diff
@@ -684,6 +684,24 @@ If the key doesn't exist, then its default value is zero:
 In other words, it's like `setvar mydict.k += 3`, but you don't have to
 initialize each value to zero.
 
+### Dict/update()
```

Comment:
I think the link from the TOC might be broken?  unless we call it update() not Dict/update()

Reply:
Fixed